src/index.html

```diff
@@ -7,12 +7,68 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Space+Grotesk:wght@300..700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file (`style.scss`). Browsers cannot directly interpret SCSS files; they need to be compiled into CSS first. Ensure that you compile your SCSS into a CSS file and link to the resulting CSS file instead.